src/journal-remote/journal-remote.c

```diff
@@ -919,6 +920,9 @@ static int remoteserver_init(RemoteServer *s,
                 else
                         url = strdupa(arg_url);
 
+                if (arg_follow)
+                        url = strjoina(url, "?follow=1");
+
```

Comment:
If we pass the follow option with the url option like --url='http://some.host:19531/entries?boot', then the code generates invalid url.